test/Renci.SshNet.IntegrationTests/SshConnectionDisruptor.cs

```diff
@@ -22,19 +22,24 @@ public SshConnectionRestorer BreakConnections()
 
         private static void PauseSshd(SshClient client)
         {
-            var command = client.CreateCommand("sudo echo 'DenyUsers sshnet' >> /etc/ssh/sshd_config");
-            var output = command.Execute();
-            if (command.ExitStatus != 0)
+            using (var command = client.CreateCommand("sudo echo 'DenyUsers sshnet' >> /etc/ssh/sshd_config"))
             {
-                throw new ApplicationException(
-                    $"Blocking user sshnet failed with exit code {command.ExitStatus}.\r\n{output}\r\n{command.Error}");
+                var output = command.Execute();
+                if (command.ExitStatus != 0)
+                {
+                    throw new ApplicationException(
+                        $"Blocking user sshnet failed with exit code {command.ExitStatus}.\r\n{output}\r\n{command.Error}");
+                }
             }
-            command = client.CreateCommand("sudo pkill -9 -U sshnet -f sshd.pam");
-            output = command.Execute();
-            if (command.ExitStatus != 0)
+
+            using (var command = client.CreateCommand("sudo pkill -9 -U sshnet -f sshd-session.pam"))
```

Comment:
sshd.pam -> sshd-session.pam is the fix for OpenSSH 9.8+. However, that release also completely removes DSA at compile time, so now the DSA tests fail.

I think it is time to remove DSA